### PR TITLE
Fix help text

### PR DESCRIPTION
**Pull request by MaryamZi**
## Purpose
$title.

Fixes https://github.com/ballerina-platform/ballerina-distribution/issues/336
Fixes https://github.com/ballerina-platform/ballerina-distribution/issues/334
Fixes https://github.com/ballerina-platform/ballerina-distribution/issues/332
Fixes https://github.com/ballerina-platform/ballerina-distribution/issues/328
Fixes https://github.com/ballerina-platform/ballerina-distribution/issues/327
Fixes https://github.com/ballerina-platform/ballerina-distribution/issues/326
Fixes https://github.com/ballerina-platform/ballerina-distribution/issues/324
Fixes https://github.com/ballerina-platform/ballerina-distribution/issues/323
Fixes https://github.com/ballerina-platform/ballerina-distribution/issues/322

### CODE REVIEW

File: ballerina-command/src/main/resources/cli-help/ballerina-help.help [hunk 1]
```diff
@@ -1,8 +0,0 @@
-
```
Comment (keizer619) [inline ballerina-command/src/main/resources/cli-help/ballerina-help.help]: @MaryamZi 
We need to keep this file as its referred by the default command. This is used to merge all the help commands together (jballerina + command)

https://github.com/ballerina-platform/ballerina-distribution/blob/release-stage/ballerina-command/src/main/java/org/ballerinalang/command/cmd/DefaultCommand.java#L51

Comment (MaryamZi) [reply ballerina-command/src/main/resources/cli-help/ballerina-help.help]: Missed that, added it back with 7a88cb7.

File: ballerina-command/src/main/resources/cli-help/ballerina-update.help [hunk 2]
```diff
@@ -1,15 +1,14 @@
...
+       ballerina-update - Update to the latest Ballerina command version
 
 SYNOPSIS
-    ballerina update
+       ballerina update
+
 
 DESCRIPTION
-     Ballerina update updates to Ballerina command to latest version.
+       Update updates the Ballerina command to the latest version.
```
Comment (Kishanthan) [inline ballerina-command/src/main/resources/cli-help/ballerina-update.help]: Shall we say, `update command updates....`

File: ballerina-command/src/main/resources/cli-help/ballerina-list.help [hunk 3]
```diff
@@ -1,12 +1,14 @@
...
+       ballerina-dist-list - List available Ballerina distributions
 
 SYNOPSIS
-    ballerina dist list
+       ballerina dist list
+
 
 DESCRIPTION
-     Ballerina dist list lists available Ballerina distributions.
+       Dist-list lists the available Ballerina distributions.
```
Comment (Kishanthan) [inline ballerina-command/src/main/resources/cli-help/ballerina-list.help]: shall we include "both locally and remotely" ?

File: ballerina-command/src/main/resources/cli-help/ballerina-use.help [hunk 4]
```diff
@@ -1,15 +1,14 @@
 NAME
-    ballerina-dist-use - Use a given distribution version
+       ballerina-dist-use - Use a given distribution version
```
Comment (Kishanthan) [inline ballerina-command/src/main/resources/cli-help/ballerina-use.help]: Shall we also include "... as the default version" ?

File: ballerina-command/src/main/resources/cli-help/ballerina-use.help [hunk 5]
```diff
@@ -1,15 +1,14 @@
...
 
 SYNOPSIS
-    ballerina dist use <distribution-version>
+       ballerina dist use <distribution-version>
+
 
 DESCRIPTION
-     Use pulls and uses a given Ballerina version from project repository.
+       Use switches to a given Ballerina version from the project repository.
```
Comment (Kishanthan) [inline ballerina-command/src/main/resources/cli-help/ballerina-use.help]: Shall we also include "... as the default version" ?

File: ballerina-command/src/main/resources/cli-help/ballerina-update.help (hunk 2 above)
Comment (MaryamZi) [reply ballerina-command/src/main/resources/cli-help/ballerina-update.help]: This was what we decided for the commands initially IINM (run, build, etc. still follows this format while there are some that have diverged).

**Pull request merged**